src/data/user-monitoring/permission-fixer/PermissionFixerUserGroupD2Repository.ts

```diff
@@ -23,21 +24,45 @@ export class PermissionFixerUserGroupD2Repository implements PermissionFixerUser
             throw new UserGroupNotFoundException("Error getting user group: " + groupsIds);
         }
     }
-    async save(userGroup: PermissionFixerUserGroupExtended): Async<string> {
+    async save(userGroup: PermissionFixerUserGroupExtended, users: Ref[]): Async<string> {
```

Comment:
You can leave it as it is, but this is not saving a userGroup, only adding members to it.